### ISSUE TITLE
Add a description of work experience that would get a pass mark to the User Guide

### ISSUE THREAD

**Issue by RachelMuston** [sara-sabr/ITStrategy#1687]
- [x] write the english text
- [x] translate
- [x] review translation
- [x] add to User Guide
- [x] Request that a link to this description be added to the Application Form (GC Forms team)

**Comment by RachelMuston** [comment]
English text: 
I did a project for my Data Concepts class in 2019 where I had to use Python and Jupyter notebooks to clean a dataset with over 25000 rows of data. I used Pandas and dataframes to drop unnecessary columns, I used Numpy to change the index and I combined Pandas str methods with NumPy’s np.where function to get the equivalent of an ‘if’ statement. This allowed me to find and fix data that was the same but not exactly the same (e.g.  in the province field were some entries had “ON”, some had “on” and some had “Ontario”. 
Or 
While I haven't used Python and Jupyter notebooks, I have done similar data cleaning using R.  In 2020 I worked on a short contract for a local non-profit where I had to clean a data set of member information. I removed duplicate entries with duplicate (), I used the gsub() function to match and replace data that was the same but not exactly the same (e.g. in the province field some entries had "ON", some had "on" and some had "Ontario", I looked for invalid values by reviewing the data of birth and ensuring no member was over.... and I used RMarkdown to clearly document all my changes.  Given my experience in R, I'm confident I would be able to pick up Python and complete this contract.


French text (needs review):
J'ai fait un projet pour mon cours de concepts de données en 2019 où je devais utiliser Python et les carnets Jupyter pour nettoyer un ensemble de données avec plus de 25000 lignes de données. J'ai utilisé Pandas et des dataframes pour supprimer les colonnes inutiles, j'ai utilisé Numpy pour modifier l'index et j'ai combiné les méthodes str de Pandas avec la fonction np.where de NumPy pour obtenir l'équivalent d'une instruction " if ". Cela m'a permis de trouver et de corriger des données qui étaient les mêmes mais pas exactement les mêmes (par exemple, dans le champ de la province, certaines entrées avaient "ON", d'autres "on" et d'autres encore "Ontario").

Traduit avec www.DeepL.com/Translator (version gratuite)
